src/kw/kw.cpp

```diff
@@ -8,16 +8,33 @@
  */
 
 #include <string>
+#include <ctime>
+
 using std::string;
 
 #include "mercury/kw/kw.h"
 
+#include <stdlib.h>
 #include <parson.h>
 #include <curl/curl.h>
 
 
+KanoWorld::KanoWorld(void)
+{
+    // Data file to store token and expiration date
+    data_filename = string(getenv("HOME")) + "/" + ".mercury_kw.json";
+    token = "";
+    expiration_date = "";
+}
+
+KanoWorld::~KanoWorld(void)
+{
+}
+
+
 /**
- * \warning Not implemented
+ *  Calls the "login" endpoint using the username and password credentials.
+ *  On success, the new token and duration time are stored in the cache data file.
  */
 bool KanoWorld::login(string username, string password, bool verbose)
```

Comment:
Pass `string`s by reference and if possible make things `const`:

```suggestion
bool KanoWorld::login(const string& username, const string& password, const bool verbose)
```